cocos/renderer/gfx-agent/BufferAgent.cpp

```diff
@@ -68,7 +68,7 @@ void BufferAgent::doInit(const BufferViewInfo &info) {
         });
 }
 
-void BufferAgent::doResize(uint size) {
+void BufferAgent::doResize(uint size, uint count) {
```

Comment:
What is the meaning of `count`?

Reply:
size / stride, passing here just to save the repetitive calculation on each backends

Reply:
Got it. It is an internal function, no problem. But it is better to add some comment of `count`.